### DISCUSSION TITLE
Re: Integration of any Road Detection algorithm into Carla

### DISCUSSION THREAD

**Discussion by bg-sumu** [carla-simulator/carla#7058] [Q&A]
I want to Integrate Yolov5 into Carla 0.9.11, can someone help me with this ? Any resources for reference would be really helpful.

**Comment by Jinviz** [comment]
You can use this example to detect an image.
https://github.com/umtclskn/Carla_Simulator_YOLOV3_Object_Detection

**Comment by bg-sumu** [comment]
Thanks for the Help, i will try doing that and get back to you.
